release-notes/opensearch-dashboards.release-notes-1.0.0.0-rc1.md

```diff
@@ -0,0 +1,20 @@
+## Version 1.0.0.0-rc1 Release Notes
```

Comment:
It does seem like this should be `1.0.0-rc1` to match the previous release version identifiers.

Reply:
It's interesting looking at the other opensearch-project release notes folders all have 1.0.0.0 which isn't following semantic versioning. 

Looking at  https://github.com/opensearch-project/opensearch-build/tree/main/release-notes because this folder has a presence on the opensearch.org website, it seems like right now it's a manual process of aggregating the release notes and then adding the contents in one file to be hosted on the website. But without knowing any tooling I'm unsure about it automation or if it's wildcarding for each repos release-notes based on the syntax "1.0.0.0-rc1". With that said should we revert it to be 1.0.0.0-rc1?

Reply:
I changed it to 1.0.0-rc1 to match opensearch.